packages/web-app-files/tests/unit/components/AppBar/CreateAndUpload.spec.ts

```diff
@@ -12,6 +12,7 @@ import {
   defaultStoreMockOptions,
   defaultComponentMocks
 } from 'web-test-helpers'
+import { RouteLocation } from 'vue-router'
```

Comment:
Rather import this from `web-test-helpers`, no?

Reply:
Good catch.

I introduced `RouteLocation` in `web-test-helpers` only to avoid having to add `vue-router` as an explicit dependency in all packages only to make `RouteLocation` available in tests. So if there's already a dependency on `vue-router` it's fine to import it from there imho.